baikal/umbrel-app.yml

```diff
@@ -0,0 +1,38 @@
+manifestVersion: 1
+id: baikal
+name: Baikal
+tagline: A lightweight CalDAV+CardDAV server
+icon: ""
```

Comment:
Sorry for the confusion here. This entire line `icon: ""` can be completely removed.